gap/attr.gi

```diff
@@ -1653,6 +1653,75 @@ function(D)
     return C;
 end);
 
+# Compute for a given rotation system the facial walks
+InstallMethod(FacialWalks, "for a digraph and a list",
+[IsDigraph, IsList],
```

Comment:
I notice that you check that `rotationSystem` is a dense list below, why not just replace:
```suggestion
[IsDigraph, IsDenseList],
```